test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/LoadingSample.cs

```diff
@@ -6,23 +6,42 @@ namespace Tizen.NUI.Samples
 {
     public class LoadingSample : IExample
     {
-        private TextLabel board1, board2, board;
-        private Button button1, button2, button3, button4;
-        private Loading loading1_1, loading2_1;  //1-null para 2-attributes; X_1-color; X_2 image track
+        private TextLabel[] textLabel = new TextLabel[3];
+        private Button[] button = new Button[3];
+        private Loading[] loading = new Loading[2];
         private View root;
+        private View gridLayout;
+        private View[] layout = new View[4];
+        private string[] imageArray;
 
         public void Activate()
         {
             Window window = NUIApplication.GetDefaultWindow();
 
             root = new View()
             {
-                Size2D = new Size2D(1920, 1080),
+                Size = new Size(1920, 1080),
+                BackgroundColor = new Color(0.7f, 0.9f, 0.8f, 1.0f)
             };
+            window.Add(root);
 
-            CreateBoardAndButtons();
-            string[] imageArray = new string[36];
-            for (int i=0; i<36; i++)
+            gridLayout = new View()
+            {
+                Position = new Position(400, 200),
+                Size = new Size(1920, 1080)
+            };
+            gridLayout.Layout = new GridLayout()
+            {
+                Columns = 2,
+                Rows = 4,
```

Comment:
GridLayout use only one property according to Orientation.
Horizontal => Row 
Vertical => Columns

Reply:
Fixed